utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt

```diff
@@ -162,6 +165,9 @@ class UrlImageParser private constructor(
             )
           proxyDrawable.initialize(drawable, computeBounds(context, drawable, viewWidth, padding))
           htmlContentTextView.text = htmlContentTextView.text
+          // This is not getting logged, it get's logged only when we go in background and
```

Comment:
This statment is not logged when we open activity for the first time (when images are not loaded)